### PR TITLE
Changing some words and translating paragraphs and descriptions (ES)

### PR DESCRIPTION
**Pull request by aerendon**
_(I hope to help you properly)_

Changing some words and translating paragraphs and descriptions
- Language: _es_

### CODE REVIEW

File: languages/es.json [hunk 1]
```diff
@@ -77,19 +87,35 @@
...
   "workshopper-count-to-6": "Aprende cómo utilizar algunas características de ES6, la proxima versión de JavaScript.",
   "workshopper-kick-off-koa": "Comienza con los primeros pasos de Koa, la próxima generación de frameworks web para Node.js.",
-  "workshopper-lololodash": "Aprende Lo-Dash (fork de underscore) para manejar tus arrays y objetos de manera muy simple",
+  "workshopper-lololodash": "Aprende Lo-Dash (fork de underscore) para manejar tus arrays y objetos de manera muy simple.",
+  "workshopper-learnyoucouchdb": "Aprende acerca de CouchDB - La base de datos que abarca completamente la web.",
+  "workshopper-learnuv": "Aprende <i>uv</i> por diversión y ganas, un taller autodirigido para una librería que alimenta a Node.js.",
+  "workshopper-learn-generators": "Una introducción a Generadores JavaScript ES6.",
+  "workshopper-learnyoureact": "¡Vamos a aprender React.js y Renderización del lado del servidor!.",
+  "workshopper-perfschool": "¡Encuentra el camino a través del laberinto de optimización de rendimiento web!.",
+  "workshopper-web-audio-school": "Aprenda sobre la API de Audio Web, completando una serie de lecciones interactivas con énfasis en música.",
```
Comment (fforres) [inline languages/es.json]: Para manter la "cercanía" al hablar con el usuario, es preferible  dejar todo "informal".

En esta caso `Aprende sobre la API...` más que `Aprenda sobre...`

File: languages/es.json [hunk 2]
```diff
@@ -50,6 +55,8 @@
   "index-get-going-editor5": ",",
   "index-get-going-editor6": ",",
   "index-get-going-editor7": ".",
+  "index-get-going-help-pre": "¿Problemas con el workshopper? Intenta instalando la última versión de Node.js (v6.2.0 o posterior), o formula una pregunta en el tablero de",
```
Comment (fforres) [inline languages/es.json]: Que tal `¿Problemas con alguno de los Workshoppers?`

**Comment by fforres** [comment]
Hi @aerendon 
Spanish speaker here, from Chile.

Thanks for the PR!
It looks really good as it is 👍 I left some messages with some small fixes, but as a whole, it's really great

**Comment by aerendon** [comment]
Hi @fforres 
I have just added your suggestions and other missing descriptions.

**Comment by fforres** [comment]
@aerendon LGTM 👍 
@martinheidegger Should we commit?

**Comment by martinheidegger** [comment]
@fforres I am focussing my time on workshopper and removed myself from the Nodeschool org since workshopper is enough work already. Feel free to discuss the decision process with @nodeschool/core members.

**Comment by riyadhalnur** [comment]
@fforres Please feel free to go ahead and merge if everything's good to go 😃

**Comment by fforres** [comment]
Merged!

Thanks @aerendon

**Pull request merged**